Frends.Salesforce.CreateSObject/Frends.Salesforce.CreateSObject.Tests/UnitTests.cs

```diff
@@ -0,0 +1,292 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using RestSharp;
+using static Frends.Salesforce.CreateSObject.Definitions.Enums;
+using JsonSerializer = System.Text.Json.JsonSerializer;
+
+namespace Frends.Salesforce.CreateSObject.Tests
+{
+    [TestClass]
+    public class UnitTests
+    {
+        private readonly string _clientSecret = Environment.GetEnvironmentVariable("Salesforce_Client_Secret");
+        private readonly string _password = Environment.GetEnvironmentVariable("Salesforce_Password");
+        private readonly string _securityToken = Environment.GetEnvironmentVariable("Salesforce_Security_Token");
+        private readonly string _clientID = Environment.GetEnvironmentVariable("Salesforce_ClientID");
+
+        private readonly string _domain = @"https://hiqfinlandoy2-dev-ed.my.salesforce.com";
+        private readonly string _username = "testuser@test.fi";
+        private readonly string _authurl = @"https://login.salesforce.com/services/oauth2/token";
+
+        private readonly CancellationToken _cancellationToken = new();
+        private Options _options;
+        private string _userJson;
+        private List<ResultObject> _result;
+
+        private string _name = "Test" + DateTime.Now.Year + "" + DateTime.Now.Month + "" + DateTime.Now.Day + "" + DateTime.Now.Hour + "" + DateTime.Now.Minute + "" + DateTime.Now.Millisecond;
+
+        #region helper classes
```

Comment:
You shouldn't require any helper classes. You should be able to run tests by using tasks own classes. If you want to use some dummy classes in tests, you can create unnamed classes using new-keyword. For example `var content = new { Name = _name }`. This will generate object with property "Name", which you can then use.